config/application.rb

```diff
@@ -58,13 +58,41 @@ class Application < Rails::Application
     # we do here instead of using initializers because of a Rails 5.1 vs
     # MySQL bug where `rake db:reset` causes ActiveRecord to be loaded
     # before initializers and causes configuration not to be respected.
-    config.load_defaults 6.1
+    config.load_defaults 7.0
+
+    # TODO: consider removing this to enable the default value 'true', and setting `allow_other_host: true` for `redirect_to` only where needed
+    # Protect from open redirect attacks in `redirect_back_or_to` and `redirect_to`.
+    config.action_controller.raise_on_open_redirects = false
+
+    # Enable parameter wrapping for JSON.
+    # Previously this was set in an initializer. It's fine to keep using that initializer if you've customized it.
+    # To disable parameter wrapping entirely, set this config to `false`.
+    config.action_controller.wrap_parameters_by_default = false
+
+    # ** Please read carefully, this must be configured in config/application.rb **
+    # Change the format of the cache entry.
+    # Changing this default means that all new cache entries added to the cache
+    # will have a different format that is not supported by Rails 6.1 applications.
+    # Only change this value after your application is fully deployed to Rails 7.0
+    # and you have no plans to rollback.
+    # When you're ready to change format, add this to `config/application.rb` (NOT this file):
+    #  config.active_support.cache_format_version = 7.0 # TODO
```

Comment:
what is the default value given we have `config.load_defaults 7.0` above? Maybe we need to set to 
```suggestion
     config.active_support.cache_format_version = 6.1 # TODO update to 7.0
```

Reply:
Yes, thank you, that's a great catch!